packages/apidom-ls/src/config/openapi/security-scheme/lint/allowed-fields-3-1.ts

```diff
@@ -0,0 +1,29 @@
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const allowedFields3_1Lint: LinterMeta = {
+  code: ApilintCodes.NOT_ALLOWED_FIELDS,
+  source: 'apilint',
+  message: 'Object includes not allowed fields',
+  severity: 1,
+  linterFunction: 'allowedFields',
+  linterParams: [
+    [
+      'type',
+      'description',
+      'name',
+      'in',
+      'scheme',
+      'bearerFormat',
+      'flows',
+      'openIdConnectUrl',
+      '$ref',
```

Comment:
```suggestion
```

Reply:
Explanation for the change in https://github.com/swagger-api/apidom/commit/d476aeffe0fe1474646dcfd4de1bb1fd39d8f151#r87246050